obsah/__init__.py

```diff
@@ -242,10 +243,26 @@ def allow_extra_vars() -> bool:
         """
         Whether to allow --extra-vars parameter to be automatically added
         """
-        if value := os.environ.get('OBSAH_ALLOW_EXTRA_VARS') is not None:
+        if (value := os.environ.get('OBSAH_ALLOW_EXTRA_VARS')) is not None:
             return value.lower() in ['true', '1']
         return True
 
+    @staticmethod
+    def persist_params():
+        """
+        Whether or not to persist parameters
+        """
+        if (value := os.environ.get('OBSAH_PERSIST_PARAMS')) is not None:
+            return value.lower() in ['true', '1']
+        return False
+
+    @staticmethod
+    def persist_path():
+        """
+        Where to persist parameters to
+        """
+        return os.environ.get('OBSAH_PERSIST_PATH', '/tmp/obsah.yaml')
```

Comment:
should this be somewhere under OBSAH_DATA? (not really, if you ask me), or should we introduce `/var/lib/obsah`?

Reply:
> we introduce `/var/lib/obsah`?

This sounds correct.

Reply:
done